### PR TITLE
Smar120 appointment(fe)

### PR DESCRIPTION
**Pull request by OlenaPosternak**
## Description changes:

- Book an appointment

## Issue ticket code (and/or) and link
[Link to JIRA ticket](https://smartdc23.atlassian.net/browse/SMAR-120)

### General
 
- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Don't have "any" on my code
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside ternary operator
- [x] Don't have commented code
- [x] All links in on ENV files
- [x] Used camelCase for variables and functions
- [x] Functions is public only if it's used outside the class
- [ ] No hardcoded values
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend
- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [ ] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

#### General view:

First step 
image

Calendar 
image

All required fields are fulfilled
image

Second step
image

Search by name  and required field is fulfilled
image

Notification 
image

### CODE REVIEW

File: src/components/Appointment/BookAppointmentForm/index.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      return;
+    }
+    setSelectedDate(day);
+
+    const appointmentDate = formatDate(day);
+    setFormattedDate(appointmentDate);
+  };
+
+  const onSubmit = (data) => {
+    console.log(data);
```
Comment (lyudad) [inline src/components/Appointment/BookAppointmentForm/index.tsx]: ?

File: src/pages/appointment/index.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Wrapper from '@components/Wrapper';
+import { PATH } from '@router/index';
+import { useTranslation } from 'react-i18next';
+import { LinkContainer } from '@components/Patient/styles';
+import BookAppointmentForm from '@components/Appointment/BookAppointmentForm';
+import PatientCardInfoShort from '@components/Appointment/PatientCardInfoShort';
+
+import { ReactComponent as ArrowLeft } from '@assets/arrowLeft.svg';
+import { BackToDashLink } from './styles';
```
Comment (lyudad) [inline src/pages/appointment/index.tsx]: here should be empty line

File: src/pages/appointment/index.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+const CreateAppointment: React.FC = () => {
+  const { t } = useTranslation();
```
Comment (lyudad) [inline src/pages/appointment/index.tsx]: here should be empty line

File: src/pages/appointment/index.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+import Wrapper from '@components/Wrapper';
+import { PATH } from '@router/index';
+import { useTranslation } from 'react-i18next';
+import { LinkContainer } from '@components/Patient/styles';
+import BookAppointmentForm from '@components/Appointment/BookAppointmentForm';
+import PatientCardInfoShort from '@components/Appointment/PatientCardInfoShort';
+
+import { ReactComponent as ArrowLeft } from '@assets/arrowLeft.svg';
+import { BackToDashLink } from './styles';
+const CreateAppointment: React.FC = () => {
```
Comment (lyudad) [inline src/pages/appointment/index.tsx:8]: here should be empty line. check your prettier confuguration

File: src/components/Appointment/BookAppointmentForm/index.tsx (hunk 1 above)
Comment (OlenaPosternak) [reply src/components/Appointment/BookAppointmentForm/index.tsx]: it's till the connection with the backend to check all data after submit. In next task here will be sending all info to backend

**Pull request merged**